docs/src/pages/customization/how-to-customize/GlobalCssOverride.js

```diff
@@ -5,7 +5,6 @@ import Button from '@material-ui/core/Button';
 const GlobalCss = withStyles({
```

Comment:
We will need to update this once we implement the <GlobalStyles /> in the `@material-ui/styled-engine`* packages